images/linux/scripts/SoftwareReport/SoftwareReport.Android.psm1

```diff
@@ -154,3 +154,9 @@ function Get-AndroidGoogleAPIsVersions {
     return ($versions -Join "<br>")
 }
 
+function Get-AndroidNDKVersions {
+    $versions = @()
```

Comment:
nitpick: don't use anywhere

Reply:
yup, you're right, was taken from macOS script